.travis.yml

```diff
@@ -13,10 +13,12 @@ notifications:
   webhooks:
     - http://savage.nonblocking.io:8080/savage/travis
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
-  - pip install urllib3[secure]
-  - pip install gsutil --user
+  - pip install --user urllib3[secure]
+  - pip install --user gsutil
+  - pip install --user protobuf
```

Comment:
nit: Is it possible to consolidate these into one line?
`pip install --user urllib3[secure] gsutil protobuf`

Could shave off half a second here :wink:

Reply:
Done.

Reply:
Whoa, the default download directory structure of `pip` is complicated, and caching it takes upwards of a minute: https://travis-ci.org/ampproject/amphtml/jobs/499909203#L443

I've pushed a new commit that does a local install from a much simpler download directory structure, that should be cached much faster. PTAL.